drivers/clock_control/Kconfig.stm32

```
@@ -121,12 +134,14 @@ config CLOCK_STM32_MCO1_SRC_HSI48
 	           SOC_SERIES_STM32H7X || \
 	           SOC_SERIES_STM32H7RSX || \
 	           SOC_SERIES_STM32H5X
+	depends on SOC_SERIES_STM32L4X || SOC_SERIES_STM32U5X
 	help
 	  Use HSI48 as source of MCO1
 
 config CLOCK_STM32_MCO1_SRC_PLLCLK
 	bool "PLLCLK"
-	depends on SOC_SERIES_STM32F4X || SOC_SERIES_STM32F7X || SOC_SERIES_STM32L4X
+	depends on SOC_SERIES_STM32F4X || SOC_SERIES_STM32F7X || SOC_SERIES_STM32L4X || \
+		   SOC_SERIES_STM32U5X
```

For consistency in the file:
```suggestion
	depends on SOC_SERIES_STM32F4X || \
	           SOC_SERIES_STM32F7X || \
	           SOC_SERIES_STM32L4X || \
		   SOC_SERIES_STM32U5X
```